### ISSUE TITLE
Audit accessibilité 75+

### ISSUE THREAD

**Issue by abulte** [etalab/data.gouv.fr#214]
- [ ] demander devis
- [ ] lancer presta
- [ ] audit préalable avec recos
- [ ] fix selon recos
- [ ] agrément 75%+

**Comment by agarrone** [comment]
@maudetes on a dépassé les 75% je peux clore ?

**Comment by maudetes** [comment]
yep, c'est bon pour moi! On pourra recréer des tickets quand on refait une passe sur ce qu'il nous reste
